Assets/Mirror/Runtime/NetworkManager.cs

```diff
@@ -442,7 +442,7 @@ void CleanupNetworkIdentities()
 
         void ClientChangeScene(string newSceneName, bool forceReload)
         {
-            ClientChangeScene(networkSceneName, forceReload, LoadSceneMode.Single, LocalPhysicsMode.None);
+            ClientChangeScene(newSceneName, forceReload, LoadSceneMode.Single, LocalPhysicsMode.None);
```

Comment:
```suggestion
            ClientChangeScene(networkSceneName, forceReload, LoadSceneMode.Single, LocalPhysicsMode.None);
```
Revert this change to avoid conflict after #866 is merged.